read.c

```diff
@@ -46,6 +46,9 @@
 #include "sds.h"
 #include "win32.h"
 
+/* The initial size of our working stack */
```

Comment:
@michael-grunder Please consider updating the comment, this is not just the initial size but also step size when growing.

Reply:
Good catch!  Updated.